frontend/src/components/FormFooter.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+export default function FormFooter({
+  isFilePicked,
+  contentFile,
+  length,
+  setFile,
+  handleSubmit,
+}) {
+  function getNiceName(requiredLength) {
+    if (requiredLength >= contentFile.name.length) {
+      return contentFile.name;
+    } else {
+      return contentFile.name.substring(0, requiredLength) + "..";
+    }
+  }
+  return (
+    <>
+      <div className="media-name-character-limit-display">
+        <div className={`${isFilePicked ? "file-selected" : "no-file"}`}>
+          <span className="media-name-full">
+            📎 {isFilePicked ? getNiceName(35) : "nothing selected"}
+          </span>
+          <span className="media-name-mobile">
+            📎 {isFilePicked ? getNiceName(16) : "nothing selected"}
```

Comment:
`Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby.`. Could we try this lint suggestion for lines 21 and 23?